examples/discovery.py

```diff
@@ -6,4 +6,5 @@ def device_discovered(self, device):
 
 manager = AnyDeviceManager(adapter_name='hci0')
 manager.start_discovery()
+manager.set_timeout(3 * 1000); # Timeout after 3 seconds
```

Comment:
I can see the value of the timeout. I would however like to see this feature as a parameter to `manager.start_discovery()`, i.e. `manager.start_discovery(timeout=3.0)` with a default to `None` (meaning no timeout).